docs/csharp/language-reference/keywords/using-directive.md

```diff
@@ -46,9 +46,15 @@ class Program
 ```  
   
 ## Remarks  
- The scope of a `using` directive is limited to the file in which it appears.  
+ The scope of a `using` directive is limited to the file in which it appears.
+ 
+ The `using` directive can appear:
+- either at the beginning of a source code file, before any namespace or type definitions;
+- or in any namespace, but before any namespace or type declared in this namespace.
+
+Otherwise, compiler error [CS1529](../../misc/cs1529.md) is generated.
   
- Create a `using` alias to make it easier to qualify an identifier to a namespace or type. The right side of a using alias directive must always be a fully-qualified type regardless of the using directives that come before it.  
+ Create a `using` alias to make it easier to qualify an identifier to a namespace or type. No alias can be used in the declaration of a using directive. The fully-qualified namespace or type must always be used regardless of the using directives that come before it.
```

Comment:
- This is unrelated to your PR, but it's best to describe "a \`using\` alias" as "a \`using\` alias directive" to be consistent with beginning of the topic.
- I'm not sure what "No alias can be used in the declaration of a using directive" means.